### ISSUE TITLE
HTML report generator

### ISSUE THREAD

**Issue by edinmilic** [IsaacWalker/PlaywrightAxeDotnet#2]
I noticed two issues:
1. The report directory property is ignored [here](https://github.com/IsaacWalker/PlaywrightAxeDotnet/blob/8ad9698cdd8980d9a85e38aa2f92c73aa6f5c464/Playwright.Axe/HtmlReport/HtmlReportBuilder.cs#L29). You probably want this string.IsNullOrEmpty(options.ReportDir) ? "report" : options.ReportDir?
2. If I do generate the html I am getting this upon opening the html file 
![image](https://user-images.githubusercontent.com/7747756/180632723-5aa76935-2503-476a-b625-657950798576.png)

Do you have any idea why that is happening?

This is my full code  
```var playwright = await PlayWright.Playwright.CreateAsync();  
 var browser = await playwright.Firefox.LaunchAsync();
var page = await(await browser.NewContextAsync()).NewPageAsync();
await page.GotoAsync("https://www.some.org/");
AxeRunOptions options = new AxeRunOptions(
    runOnly: new AxeRunOnly(AxeRunOnlyType.Tags, new List<string> { "wcag2aa" }),
    resultTypes: new List<AxeResultGroup>()
    {
        AxeResultGroup.Violations
    }
);
var results = await page.RunAxe(options, reportOptions: new AxeHtmlReportOptions("C:\\myReport", false));

**Comment by IsaacWalker** [comment]
I think the second issue was caused by a difference between the nullability of the Targets for the axe-core types and the C# equivalents.

Both issues should be fixed now in [1.3.0](https://www.nuget.org/packages/Playwright.Axe/1.3.0).

**Comment by edinmilic** [comment]
I have confirmed that the 1.3.0 has fixed both issues. Thanks for the fix and good work on the library!

Don't know if you are aware of this library that specifically builds html report that looks like this https://lpelypenko.github.io/axe-html-reporter/

The package is hosted here https://github.com/lpelypenko/axe-html-reporter

The report this library generates might be a bit more comprehensive and actionable.
Cheers!
